admissioncontroller/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
no need. it can stay under the apache-2 license of the parent repo

Reply:
Great!